pkg/cmd/roachtest/cluster.go

```diff
@@ -1278,6 +1278,7 @@ WHERE t.status NOT IN ('RANGE_CONSISTENT', 'RANGE_INDETERMINATE')`)
 		l.Printf("consistency check failed with %v; ignoring", err)
 		return nil
 	}
+	defer rows.Close()
```

Comment:
Nice. A quick check using the sqlclosecheck shows we have a few more cases of omitting a Close().  In many cases it likely doesn't matter because we are following up with t.Fatal(). (No need to handle them here, I'll open an issue)